webapp/store/country/countrySlice.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-param-reassign */
```

Comment:
is this needed? plus, do not add it at the beginning of the file.
just where it's needed